README.md

```diff
@@ -1,2 +1,108 @@
-# doppler-html-editor-api
-API Rest from Doppler HTML Editor
+# Doppler HTML Editor API
+
+## Campaigns
+`GET /accounts/{accountName}/campaigns/{campaignId}/content/design`
```

Comment:
In Markdown, try to always use an interline between different paragraphs

```suggestion
## Campaigns

`GET /accounts/{accountName}/campaigns/{campaignId}/content/design`
```

Reply:
Fixed!